src/CMakeLists.txt

```diff
@@ -171,6 +171,10 @@ target_include_directories(digidocpp_priv PUBLIC
     ${XSD_INCLUDE_DIR}
 )
 
+target_link_libraries(digidocpp_util
```

Comment:
I think you are missing some conditions here, thats why IOS builds are failing

Reply:
I've added the condition used in the other hunk, let's see what CI says.